x-pack/solutions/security/plugins/cloud_defend/public/components/control_general_view_response/index.tsx

```diff
@@ -61,7 +60,6 @@ export const ControlGeneralViewResponse = ({
   const [isPopoverOpen, setPopoverOpen] = useState(false);
   const styles = useStyles();
   const selectorStyles = useSelectorStyles();
-  const visColorsBehindText = euiPaletteColorBlindBehindText();
```

Comment:
behind text colors won't be available in Borealis. Even though cloud_defend is being deprecated, replacing for safety to a similar semantic tokens